### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by ashumz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,62 @@
 Waffle.io  [![Stories in Progress](https://badge.waffle.io/waffleio/waffle.io.svg?label=waffle%3Ain%20progress&title=In%20Progress)](http://waffle.io/waffleio/waffle.io)
 =========
+### Waffle is the tastiest way to plan, organize, and track your work in GitHub & GitHub Issues. Sign up free at [http://waffle.io](http://waffle.io)
```
Comment (homeyer) [inline README.md]: `https`

File: README.md [hunk 2]
```diff
@@ -1,32 +1,62 @@
...
 
-A public repo to keep track of our current work.
+### About this repo
 
-Waffle listens to the cues in your process to update your work status automatically. Waffle.io is integrated with GitHub and allows you to view issues related to a particular repository. It allows you to create your own columns of tasks and visualize the workflow of the repository.
+This is the Waffle team's Issue Only repository for public facing feature work, enhancements, and issues.
```
Comment (homeyer) [inline README.md]: want to put "Issue Only" in quotes?

File: README.md [hunk 3]
```diff
@@ -1,32 +1,62 @@
...
 =========
+### Waffle is the tastiest way to plan, organize, and track your work in GitHub & GitHub Issues. Sign up free at [http://waffle.io](http://waffle.io)
 
-A public repo to keep track of our current work.
+### About this repo
 
-Waffle listens to the cues in your process to update your work status automatically. Waffle.io is integrated with GitHub and allows you to view issues related to a particular repository. It allows you to create your own columns of tasks and visualize the workflow of the repository.
+This is the Waffle team's Issue Only repository for public facing feature work, enhancements, and issues. 
+
+Waffle users can add issues through the GitHub interface or directly on our [Waffle.io board](http://waffle.io/waffleio/waffle.io).
```
Comment (homeyer) [inline README.md]: `https`

File: README.md [hunk 4]
```diff
@@ -1,32 +1,62 @@
...
+
+Sick of project management tools with unreliable GitHub Integrations? Waffle is built on top of the GitHub API, so your work and its status will always stay in sync. All you need to do is sign in to Waffle using GitHub. Within 5 minutes, you can transform your long lists of GitHub Issues and Pull Requests into a big picture view of what's planned, what's being worked on, and what's up next. 
+
+#### Pick your favorite view: Cards on a Board in Waffle or Issues in GitHub
+
+When you create a project in Waffle, your Issues and Pull Requests transform into cards on a board. The friendly board view, similar to Trello, is built with everyone on your team in mind. Developers can still use GitHub for development-heavy tasks like code reviews.
+
+#### Maintain a single source of truth in your workflow
+
+Add, update, comment on, and close cards in GitHub, or in Waffle. View all updates in real time, regardless of where you do your work. Always know  what's planned, in progress, waiting for review, and ready to ship.
```
Comment (homeyer) [inline README.md]: there's an extra space in `Always know  what's planned`

File: README.md [hunk 5]
```diff
@@ -1,32 +1,62 @@
...
+
+#### Track the work you're already doing with ease
+
+Track your work *as you work.* Updating your status from GitHub is simple-- just add a label to the issue. Use cues in commits or pull requests to move cards across the board automatically. Move work in progress with the push of a branch. For team members who prefer Waffle’s board view, updating status is as easy as dragging and dropping cards across the board.
```
Comment (homeyer) [inline README.md]: want to link out to the FAQ for auto work tracking?

**Comment by burtonjc** [comment]
Can we squash all these commits before merging please? `git rebase -i HEAD~[number of commits]`

**Comment by homeyer** [comment]
Squashed it for you @ashumz :). Merge when you're ready! :+1:

**Pull request merged**